ipatests/test_integration/test_installation.py

```diff
@@ -1916,3 +1917,70 @@ def test_replica0_install(self):
         tasks.install_replica(
             self.master, self.replicas[0], setup_ca=False, setup_dns=False
         )
+
+
+class TestInstallwithSHA384withRSA(IntegrationTest):
+    num_replicas = 0
+
+    @classmethod
+    def install(cls, mh):
+        pass
+
+    def test_install_master_withalgo_sha384withrsa(self, server_cleanup):
+        tasks.install_master(
+            self.master,
+            extra_args=['--ca-signing-algorithm=SHA384withRSA'],
+        )
+
+        # check Signing Algorithm post installation
+        dashed_domain = self.master.domain.realm.replace(".", '-')
+        cmd_args = ['certutil', '-L', '-d',
```

Comment:
Why not use `openssl x509 -in /var/lib/ipa/ra-agent.pem -noout -text` here too?

Reply:
`openssl` can be used. I thought of checking two different certs in two tests. But I can change it to check for RA Agent cert Signature Algorithm

Reply:
I don't think it will be needed, since you check the RA Agent during the second test.